irobotclient/configuration_handler.py

```diff
@@ -0,0 +1,159 @@
+"""
+Copyright (c) 2017 Genome Research Ltd.
+
+This program is free software: you can redistribute it and/or modify it
+under the terms of the GNU General Public License as published by the
+Free Software Foundation, either version 3 of the License, or (at your
+option) any later version.
+
+This program is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with this program. If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import argparse
+import string
+import re
+import os
+import errno
+
+from irobotclient.custom_exceptions import IrobotClientException
+
+
+def _get_command_line_agrs():
```

Comment:
Good spot.  It's only persisted so long because I'm working in an IDE and using suggestive typing.  If I was developing in the terminal this would have been a massive pain in the butt.